src/main/java/com/robin/hoteless/model/Reservation.java

```diff
@@ -0,0 +1,119 @@
+package com.robin.hoteless.model;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    private int id;
+    private Date reservationCreationDate;
+    private int lengthOfStay;
+    private double dailyRate;
+    private LocalDate checkInDate;
+    private LocalDate checkOutDate;
+    private String roomType;
+    private double totalCostOfStay;
+    private static int totalReservations;
```

Comment:
This is a confusing name because it implies recursion (self within self); perhaps it would be totalPersons? Or roomOccupants?